tasks/backend.py

```diff
@@ -48,17 +48,12 @@ def format_all(context: Context):
 # Testing tasks
 # ----------------------------------------------------------------------------
 @task
-def ruff(context: Context, docker: bool = False):
```

Comment:
Probably best to leave the option to run it in docker for now

Reply:
Why?